packages/@vuepress/theme-default/util/index.js

```diff
@@ -54,6 +54,13 @@ export function isActive (route, path) {
 }
 
 export function resolvePage (pages, rawPath, base) {
+  if (isExternal(rawPath)) {
+    return {
+      type: 'external',
+      title: rawPath,
```

Comment:
It looks strange to assign `rawPath` to `title`, you can omit it.

Reply:
Good point. Done